src/patternfly/components/Switch/switch.scss

```diff
@@ -139,9 +140,9 @@
 
   .pf-c-switch__toggle-icon {
     position: absolute;
-    top: var(--pf-c-switch__toggle-icon--Top);
```

Comment:
If you're going to remove `top`, there's no need to leave the variable around since it would not do anything.

Instead of `line-height` you could use `top: 50%; left: 0; transform: translateY(50%);` to center it. I think that would be more reliable.

Also looks like we need some more space between this icon and the circle. 

I checked out your branch to work on this a bit and created a PR - https://github.com/christiemolloy/patternfly-next/pull/17

Made some of these changes, kept this `top` and removed `line-height` below. Also reduced the size of the toggle icon - it's 14px on master currently, so we can keep it that width. Only thing I'm not sure about is I hardcoded a `2px` spacer into the switch `width` and toggle circle `translateX` to match what's on master now and create space between the toggle icon (check) and toggle circle. Feel free to update that if you can think of a better way.

Reply:
Thanks for adding that branch, I also updated some things.
- for the toggle::before width I subtracted 2px because the current toggle was 2px too big from what was on master:

<img width="178" alt="Screen Shot 2019-07-16 at 9 46 56 AM" src="https://user-images.githubusercontent.com/20118816/61304866-d8395300-a7b7-11e9-9bca-cb0a33be6f11.png">
<img width="174" alt="Screen Shot 2019-07-16 at 9 47 01 AM" src="https://user-images.githubusercontent.com/20118816/61304867-d8395300-a7b7-11e9-9661-2e02bc3ef0db.png">

- removed the var you added for transform because it looks like it isn't needed
- i think we can remove the font-weight and line-height at the next breaking change
- since we use #{pf-size-prem(2px)} 3 times now, I wonder if we should store it in a var like "offset" or something

Reply:
> since we use #{pf-size-prem(2px)} 3 times now, I wonder if we should store it in a var like "offset" or something

Works for me!